set.mm

```diff
@@ -171599,11 +171599,10 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) ) ~~> x ) \/
        22-Apr-2014.) $)
     fz1f1o $p |- ( A e. Fin -> ( A = (/) \/ ( ( # ` A ) e. NN /\
                    E. f f : ( 1 ... ( # ` A ) ) -1-1-onto-> A ) ) ) $=
-      ( cfn wcel chash cfv cc0 wceq cn wo c0 c1 cfz co cv wf1o wex sylib mpbid
-      wa cn0 hashcl orcomd hasheq0 cen wbr hashfz1 syl wb fzfi hashen mpan bren
-      elnn0 biantrud orbi12d ) ACDZAEFZGHZURIDZJAKHZUTLURMNZABOPBQZTZJUQUTUSUQU
-      RUADZUTUSJAUBZURUNRUCUQUSVAUTVDACUDUQVCUTUQVBAUEUFZVCUQVBEFURHZVGUQVEVHVF
-      URUGUHVBCDUQVHVGUILURUJVBAUKULSVBABUMRUOUPS $.
+      ( cfn wcel chash cfv cc0 wceq cn wo c0 c1 cfz co cv wf1o wex cn0 hashcl
+      wa elnn0 sylib orcomd hasheq0 cen wbr isfinite4 bren sylbb biantrud mpbid
+      orbi12d ) ACDZAEFZGHZUNIDZJAKHZUPLUNMNZABOPBQZTZJUMUPUOUMUNRDUPUOJASUNUAU
+      BUCUMUOUQUPUTACUDUMUSUPUMURAUEUFUSAUGURABUHUIUJULUK $.
```

Comment:
Feel free to add the "Shortened by" contribution tag!

Reply:
I realize it is an option, but don't put it in for my sake.

I don't know if the level of effort is a huge factor, but in this case all that happened was that the minimizer was capable of using `isfinite4im` (from iset.mm) but not `isfinite4` (from set.mm) because it would have needed to insert a `biimpi` or something of the sort. So what I did was only one step removed from running the minimizer, which by longstanding practice does not result in a "Shortened by" tag.

Reply:
I see. Anyway, good catch!